src/Push/Registry/ResponseCollection.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Wearesho\Bobra\Ubki\Push\Registry;
+
+use Carbon\Carbon;
+
+use Wearesho\Bobra\Ubki\Push\Registry;
+
+/**
+ * Class ResponseCollection
+ *
+ * @package Wearesho\Bobra\Ubki\Push\Registry
+ */
+class ResponseCollection extends \ArrayObject implements \JsonSerializable
```

Comment:
Collection have responsibility of storing items.
Creating items isn't purpose of collections